components/raftstore/src/store/fsm/store.rs

```diff
@@ -1522,6 +1525,13 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> StoreFsmDelegate<'a, EK, ER
 
     fn on_raft_message(&mut self, mut msg: RaftMessage) -> Result<()> {
         let region_id = msg.get_region_id();
+        let msg_type = msg.get_message().get_msg_type();
```

Comment:
This place is wrong. The correct place is `PeerFsmDelegate::on_raft_message`.

Reply:
PeerFsmDelegate has been dealed with. not sure now about store delegate, but will router.send()， ignore maybe ok